tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -206,3 +228,36 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
         assert session.computeresource.vm_status(
             name, rhev_data['vm_name']) is not status
+
+
+@tier2
+@parametrize('provider', _provider_parameteres)
+def test_positive_create_edit_desc_delete_resource(session, module_org, module_loc, provider):
+    """Edit VMWare Compute Resource with another description
```

Comment:
doc string is invalid as you work not only with VMWare
and
expected result is also need to be changed

Reply:
Oh, right, will change.